src/v2/guide/accessibility.md

```diff
@@ -0,0 +1,301 @@
+
+## Basic accessibility
+
+Accessibility in standard websites, such as putting alt text on images and making sure your text is large enough to read, has been covered many times before. Instead of rewriting what has already been said, here's a few resources to get you started:
+
+### Talks and Videos
+- [W3C Web Accessibility Initiative's "Introduction to Web Accessibility and W3C Standards"][W3C Intro]
+- [Leonie Watson speaking on "Developer's Guide to Accessibility Mechanics"][Leonie A11y Mechanics]
+- [Marcy Sutton speaking on "Innovating with Accessibility in Mind"][Marcy Innovating A11y]
+- [Google's "A11ycasts" series with Rob Dodson][A11ycasts]
+
+### Blogs and How-Tos
+- [W3C's Web Accessibility Tutorials][W3C Tutorials]
+- [The A11y Project][A11y Project]
+- [Heydon Pickering's Inclusive Components][Inclusive Components]
+
+## Accessibility in JavaScript-rich websites
+
+JavaScript allows for powerful, interactive websites and applications. Its usage can be simple or complex. Unfortunately, some assistive technologies struggle to support this environment. With this in mind, what can we do to accommodate these tools added needs? 
+
+In addition to the guidelines in the articles above and in the [Web Content Accessibility Guidelines (WCAG)][WCAG 2.0], we have to take further measures to make sure that by using JavaScript, we're not making our websites inaccessible to screen reader users.
+
+### Mouse-only content
+
+There's a significant number of people browsing the web without a mouse—they could be using a screen reader, or they could be using just a keyboard. It's important to make sure that there are no parts of your website that require the use of the mouse to access.
+
+Some common examples of parts of sites that can't be access without a mouse include:
+
+- Buttons or links that aren't using semantic elements or aria roles.
+- Menu dropdowns that appear when you mouse over a parent menu item.
+- Autocomplete dropdowns and search suggestions that hide when the input element loses focus.
+- Controls that appear on mouseover.
+- Anything else that relies on the mouse hovering over anything.
+
+@todo: bad example
+
+It's better to make content display on click instead of on hover - or both. For example, a menu that appears on hover and closes when the mouse moves away could continue doing that, but also open or close when clicked, whether with the mouse or the keyboard.
+
+@todo: good example
+
+### Extended input elements
+
+Standard input elements such as text inputs (<input type="text" placeholder="Example text input">) and checkboxes (<input type="checkbox">) often don't look great, so it's fairly common to want to change the appearance of them.
```

Comment:
```suggestion
Standard input elements such as text inputs (`<input type="text" placeholder="Example text input">`) and checkboxes (`<input type="checkbox">`) often don't look great, so it's fairly common to want to change the appearance of them.
```

The html code should be wrapped by a backquote, or it wouldn't be displayed well.